components/tools/OmeroJava/test/integration/gateway/SearchFacilityTest.java

```diff
@@ -71,7 +71,7 @@ protected void setUp() throws Exception {
         super.setUp();
         initData();
         // wait a little bit for the search indexer
-        Thread.sleep(10000);
+        Thread.sleep(30000);
```

Comment:
Do we know if that helped? Curious that the indexer might have taken more than 10s to index a bit of new data.

Reply:
I could set it to 10s again and see what happens?

Reply:
I'll open a PR for 5s to really test it. :smiley:

Reply:
Now opened #5959.